src/lib/coaler/embedder/ConformerEmbedder.cpp

```diff
@@ -5,15 +5,60 @@
 #include "ConformerEmbedder.hpp"
 
 #include <GraphMol/DistGeomHelpers/Embedder.h>
+#include <GraphMol/FMCS/FMCS.h>
+#include <GraphMol/ForceFieldHelpers/MMFF/MMFF.h>
 #include <GraphMol/ForceFieldHelpers/UFF/UFF.h>
 #include <GraphMol/MolAlign/AlignMolecules.h>
 #include <GraphMol/SmilesParse/SmartsWrite.h>
+#include <GraphMol/SmilesParse/SmilesWrite.h>
 #include <GraphMol/Substruct/SubstructMatch.h>
 #include <spdlog/spdlog.h>
 
+#include <utility>
+
+#include "../core/Matcher.hpp"
+#include "../multialign/Forward.hpp"
+#include "../multialign/models/Forward.hpp"
+#include "Forward.hpp"
+#include "GraphMol/FMCS/FMCS.h"
+
 const unsigned seed = 42;
 const float forceTol = 0.0135;
 
+namespace {
+    auto get_mcs(const RDKit::MOL_SPTR_VECT &mols, bool strict = true) {
```

Comment:
would be cool if this could stay in the core matcher smh. maybe as a static method and name it properly. So the logic for mcs is in one place